edxval/serializers.py

```diff
@@ -193,3 +193,20 @@ def update(self, instance, validated_data):
                 VideoImage.create_or_update(course_video, image_name)
 
         return instance
+
+
+class TranscriptPreferenceSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Transcript Preference object
+    """
+    class Meta:  # pylint: disable=C1001, C0111
+        model = TranscriptPreference
+        fields = (
```

Comment:
We should remove `fields` option as this would give error on serializer : `AssertionError: Cannot set both 'fields' and 'exclude' options on serializer TranscriptPreferenceSerializer.`